src/System/Taffybar/Information/Brightness.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE OverloadedStrings #-}
+module System.Taffybar.Information.Brightness (readBrightnessValues) where
+
+import System.UDev.Context
+import System.UDev.Device
+import System.UDev.Monitor
+
+import System.UDev.Enumerate
+import System.UDev.List
+
+import Control.Concurrent (Chan, newChan, writeChan, threadWaitRead)
+
+import Control.Monad (when)
+
+import Data.ByteString (ByteString())
+
+import System.Taffybar.Util (backgroundLoop)
+import System.Taffybar.Util.ReadByteString (parseInt)
+
+--Only returns the first match
+scanForDeviceBySubsystem :: UDev -> Subsystem -> IO Device
+scanForDeviceBySubsystem udev subSysName = do
+  e <- newEnumerate udev
```

Comment:
What is e? I can't tell from reading this because there is no type annotation. Generally I'd prefer slightly longer somewhat descriptive names

Reply:
It's essentially a C pointer, I guess: http://hackage.haskell.org/package/udev-0.1.0.0/docs/System-UDev-Enumerate.html#t:Enumerate
Kind of want to make reader-monad type things for these things to run in (all the following lines require that `e`)? Something like that could be upstreamed in the udev lib.  Though maybe I should go with the straightforward explicit stuff for now.